icu4j/main/classes/core/src/com/ibm/icu/impl/breakiter/BurmeseBreakEngine.java

```diff
@@ -31,23 +31,22 @@
     private static final byte BURMESE_MIN_WORD = 2;
 
     private DictionaryMatcher fDictionary;
-    private static UnicodeSet fBurmeseWordSet;
-    private static UnicodeSet fEndWordSet;
-    private static UnicodeSet fBeginWordSet;
-    private static UnicodeSet fMarkSet;
+    private UnicodeSet fEndWordSet;
+    private UnicodeSet fBeginWordSet;
+    private UnicodeSet fMarkSet;
 
-    static {
+    public BurmeseBreakEngine() throws IOException {
         // Initialize UnicodeSets
-        fBurmeseWordSet = new UnicodeSet();
+        UnicodeSet burmeseWordSet = new UnicodeSet();
         fMarkSet = new UnicodeSet();
         fBeginWordSet = new UnicodeSet();
 
-        fBurmeseWordSet.applyPattern("[[:Mymr:]&[:LineBreak=SA:]]");
-        fBurmeseWordSet.compact();
+        burmeseWordSet.applyPattern("[[:Mymr:]&[:LineBreak=SA:]]");
```

Comment:
Optional: You can replace `new UnicodeSet()` + `applyPattern(pattern)` with `new UnicodeSet(pattern)`.